docs/source/contributor-guide/governance.md

```diff
@@ -27,7 +27,12 @@ The current PMC and committers are listed in the [Apache Phonebook].
 
 DataFusion is part of the [Apache Software Foundation] and is governed following
 the [Apache Way] and [project management guidelines], [independently of
-commercial interests].
+commercial interests]. That means that the DataFusion project manages code for
+the benefit of all users, and not just for some company or vendor. In
+particular, DataFusion only recognizes individuals as committers never
+companies.[^1]
```

Comment:
A lot of contributors/committers are working with companies/vendors and this statement might be controversial?

Reply:
Maybe this would be better phrased as "not just for a **single** company or vendor" 🤔 

I think it would also be ok to close this PR and simple leave the link to https://community.apache.org/projectIndependence.html#apache-projects-are-managed-independently rather than trying to summarize the nuance here